lib/trento/domain/sap_system/sap_system.ex

```diff
@@ -642,4 +572,129 @@ defmodule Trento.Domain.SapSystem do
         false
     end)
   end
+
+  defp abap_or_messageserver?(features) do
```

Comment:
Do we still need these 2 functions? I think the de-registration code added exactly the same functionality:
https://github.com/trento-project/web/blob/deregistration/lib/trento/domain/sap_system/sap_system.ex#L720

Reply:
I want to leave this for now and then maybe remove when they are merged together